### ISSUE TITLE
Exporting bindings not working

### ISSUE THREAD

**Issue by orchardbot** [OrchardCMS/Orchard#3275]
**OlaK** created:
https://orchard.codeplex.com/workitem/19447

I'm having problems exporting (via the Export module) a binding I've created through the Bindings tab on the Query page, has anyone come across this before? 

I tried first on my project I'm working on, when that didn't work, I set up a clean install of Orchard 1.6 and tried there. However the binding does not appear in the exported xml, the query I created pointing at the binding comes out ok and is still pointing to the type. 

I checked in the error log but there's nothing in there... 

I'm attaching the exported xml, see the query with title "Test binding Author Export Query".

**Comment by jersiovic** [comment]
I have updated the @StanleyGoldman module to be supported on Orchard 1.10
https://github.com/xkproject/Orchard-ImportExportBindings/tree/1.10.x_XK
Íf you want I can do a pull request as it is (in a separate module) or integrated in Orchard.Projections that is its natural location.

**Comment by dukeofdarkness23** [comment]
Further to @jersiovic commits, I found that re-running an import created duplicate bindings - because there is no unique key on the MemberBinding table and no checks for duplicates in the import step. I added code in my own branch that checks for duplicates based on an assumption that the 3 columns (Type/Member/DisplayName) should correspond to a unique key and either updates or inserts accordingly.

[https://github.com/BusinessIntegrations/Orchard/tree/BI/issue/122-B](https://github.com/BusinessIntegrations/Orchard/tree/BI/issue/122-B)

Open question as to which columns should be treated as a key (Type and Member as a minimum) - I've used DisplayName purely for cases where site admins would manage queries and expect to see a contextual display of what that column binding represents to them rather than the technical property name.
